src/helpers/isHotPost.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  minimumLikes,
+  minimumNumberOfComments,
+  minimumParticipants,
+  minimumViews,
+  relevancePeriod,
+} from '@/data/hotPost'
+import getFeedsContract from '@/helpers/getFeedsContract'
+import getViews from '@/helpers/postViews'
+
+async function getParticipants(
+  feedId: number,
+  postId: number,
+  skip = 1,
+  pinned = false
+) {
+  const [{ participants }] = await getFeedsContract.getPostsAndParticipants(
+    feedId,
+    postId,
+    skip,
+    pinned
+  )
+
+  return participants.length
+}
+
+async function getNumberOfComments(feedId: number, postId: number) {
+  const lastCommentId = await getFeedsContract.lastCommentIds(feedId, postId)
+  return lastCommentId.toNumber()
+}
+
+async function getNumberOfLikes(feedId: number, postId: number) {
+  const [, likes] = await getFeedsContract.getReactions(feedId, postId, 0)
```

Comment:
Not `likes`, it's `upVotes`, let's be consistent

Reply:
If we'll introduce other reaction types, `likes` will refer to something like 👍 , instead of arrow-up